DoSomething/gambit
# Adds favicon

#### What's this PR do?
Adds a favicon.

#### How should this be reviewed?
* Pull down locally and verify http://ocalhost:5000/favicon.ico returns favicon
* Verify http://ds-mdata-responder-staging.herokuapp.com/favicon.ico returns favicon (I've deployed this branch to staging to test)

#### Any background context you want to provide?
We turned on error reporting for 404's in New Relic to know when there's missing entities when we query Contentful.... but Slack and other apps are looking for our favicon, which triggers 404's we don't want to pay attention to. 

<img width="408" alt="screen shot 2017-05-10 at 9 46 38 am" src="https://cloud.githubusercontent.com/assets/1236811/25910573/a03ecd16-3565-11e7-8d62-9b6cea924dfa.png">


#### Checklist
